packages/block-editor/src/store/actions.js

```diff
@@ -292,17 +380,39 @@ export function insertBlock( block, index, rootClientId, updateSelection = true
  * @param {?string}  rootClientId    Optional root client ID of block list on which to insert.
  * @param {?boolean} updateSelection If true block selection will be updated.  If false, block selection will not change. Defaults to true.
  *
- * @return {Object} Action object.
- */
-export function insertBlocks( blocks, index, rootClientId, updateSelection = true ) {
-	return {
-		type: 'INSERT_BLOCKS',
-		blocks: castArray( blocks ),
-		index,
-		rootClientId,
-		time: Date.now(),
-		updateSelection,
-	};
+ *  @return {Object} Action object.
+ */
+export function* insertBlocks(
+	blocks,
+	index,
+	rootClientId,
+	updateSelection = true
+) {
+	blocks = castArray( blocks );
+	const allowedBlocks = [];
+	for ( const block of blocks ) {
+		if ( block ) {
```

Comment:
Why are there falsey entries in `blocks` ?

Reply:
I guess it is never falsey, I updated the code to remove this check.